lightwood/encoder/time_series/rnn.py

```diff
@@ -23,12 +23,11 @@
 class TimeSeriesEncoder(BaseEncoder):
 
     def __init__(self, stop_after: int, is_target=False, original_type: str = None, target: str = None,
-                 grouped_by: List[str] = []):
+                 grouped_by: List[str] = [], encoder_type='rnn'):
```

Comment:
I'd kinda feel much better if the transformer based encoder as a separate encoder, but if this removes a lot of boilerplate compare to having tow separate ones, fine